scripts/templates/component-story.js

```diff
@@ -1,44 +1,39 @@
 module.exports = componentName => [
   {
-    content: `// Generated with scripts/create-componnent.js
-import {
-  AutoExample,
-  Tab,
-  StoryContainer,
-  Title,
-  Example,
-} from '@catho/quantum-storybook-ui';
+    content: `
```

Comment:
Why did you remove the "// Generated with scripts/create-componnent.js" instead of adding it?